Update Sharp Aquos TV configuration to new style

dnguyen800 opened the pull request:
**Description:**
Updates Sharp Aquos TV configuration variables to new style, as per #6385.


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

dnguyen800 committed:
Update Sharp Aquos TV configuration to new style

Updates Sharp Aquos TV configuration variables to new style, as per #6385.

fabaff commented on source/_components/media_player.aquostv.markdown:
```diff
@@ -27,14 +27,37 @@ media_player:
+  required: true
+  type: string
+port:
+  description: The port of the Sharp Aquos TV. Defaults to 10002.
```
Please remove `Defaults ...`.

fabaff commented on source/_components/media_player.aquostv.markdown:
```diff
@@ -27,14 +27,37 @@ media_player:
+  default: 10002
+  type: int
+username:
+  description: The username of the Sharp Aquos TV. Defaults to admin.
```
Dito

fabaff commented on source/_components/media_player.aquostv.markdown:
```diff
@@ -27,14 +27,37 @@ media_player:
+  default: admin
+  type: string
+password:
+  description: The password of the Sharp Aquos TV. Defaults to password.
```
Dito

fabaff commented on source/_components/media_player.aquostv.markdown:
```diff
@@ -27,14 +27,37 @@ media_player:
+  required: false
+  type: string
+power_on_enabled:
+  description: If you want to be able to turn on your TV. Defaults to False.
```
Dito

dnguyen800 committed:
Remove default desc from Sharp Aquos configuration

Removed mention of defaults from the description in configuration variables since default variable is listed.

ludeeus commented on source/_components/media_player.aquostv.markdown:
```diff
@@ -27,14 +27,37 @@ media_player:
+  description: If you want to be able to turn on your TV.
+  required: false
+  default: false
+  type: string
```
This should be `boolean`

dnguyen800 replied:
I thought so too, but I looked at the Home Assistant documentation and the allowed entries are: string, int, time, template or map. So I went with string.

https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

ludeeus replied:
@dnguyen800 from that link:
```
type:: The type of the variable. Allowed entries: boolean, string, integer, float, time, template, device_class, icon or map/list (for a list of entries). For multiple possibilities use [string, integer]. If you use map/list then should define keys: (see the template sensor for an example). If you use boolean, then default: must be defined.
```
:)

dnguyen800 replied:
haha, that's funny! I am 100% sure it did not show boolean, float, device_class, or icon when I viewed that site yesterday. Anyways, I'll change it to boolean right now.

dnguyen800 committed:
Change power_on_enabled config variable type to boolean

Changed power_on_enabled configuration variable type to boolean from string.
